sixtyfps_compiler/fileaccess.rs

```diff
@@ -22,6 +22,10 @@ impl<'a> VirtualFile<'a> {
     }
 }
 
+pub fn styles() -> Vec<&'static str> {
+    builtin_library::styles()
```

Comment:
On second thought, isn't this function perhaps simply just like this?

```rust
pub fn styles() -> impl Iterator<Item = &'static str> {
    builtin_library::widget_library().iter().map(|(style, _)| *style)
}
```

Reply:
i think we don't want to list "common".
but if we filter that out, it might make sense, yes.

Reply:
> I stick with an static array of static strings:-)

But here you're returning a Vec.  A static array of string would be better (`&'static [&'static str]`)

Reply:
also, this looks like this could be `pub use builtin_library::styles;`